sputnikdao-factory2/src/lib.rs

```diff
@@ -138,6 +144,109 @@ impl SputnikDAOFactory {
             .update_contract(account_id, code_hash, "update");
     }
 
+    /// Allows a DAO to store the official factory version as a blob, funded by the DAO wanting to upgrade
+    /// Required to successfully upgrade a DAO via proposals (proposal to store blob, proposal to upgrade from local blob)
+    /// Only intended for sputnik v2 DAO's created by sputnik factory
+    /// Payment is needed to cover storage costs for code blob size, paid by the DAO and returned upon blob removal
+    /// See docs for usage reference
```

Comment:
What docs are you referring to? I would remove this line, the description you've provided is crystal clear :D